docu/files/Multi-tabEditor.html.md

```diff
@@ -0,0 +1,96 @@
+---
+title: 'The multi-tab editor component in the Builder'
+service: 'Builder SDK'
+area: 'Core'
+type: Tutorial
+order: 4
+---
+
+The multi-tab editor is a custom Angular directive in **builder_editors** with a dependency to the **builder**.
+
+### Configure the multi-tab editor
+
+As a Builder module developer, you need to complete a few simple tasks:
+
+* Include the https://builder.yaas.io/public/js/builder_editors_v3.js file. Add **builder_editors** to your module.
+* The directive expects data formatted as in this example:
+
+```js
+    /*simple string with localized property label */
+    $scope.localizedLabel = 'Demo Label';
+    
+    $scope.configuration = {
+    'en': {'default': true, 'required': true},
+    'de': {'default': false, 'required': false}
+    };
+
+```
+Specify the data that you would like to display in the element. The multi-tab editor also accepts localized data formatted as in this example:
+
+```js
+    /*json object containing language iso and localized value of a property*/
+    $scope.localizedData = {
+        'en': 'english text',
+        'de': 'german text',
+        'pl': 'polish text'};
+```
+Remember that a list of languages (currencies) in which the property is localized has to match the list of defined languages (currencies) for the current user.
+* HTML markup
+```html
+    <div class="panel-body">
+        <h3>Single Line</h3>
+        <builder-tab-editor builder-locales="configuration" data="localizedData" ng-form="someName2" ></builder-tab-editor>
+    </div>
+```
+
+#### Attributes: 
+
+- <b>data</b> - Required. Indicates which text should fill the editor prior to user input. 
+- <b>builder-validate-tabs</b> - Not required. Default false. Specifies whether a field should be validated or is required in order to save.
+- <b>builder-locales</b> - A structure specifing which locale is default and/or must be filled. This attribute enables you to choose between a multi-tab editor for languages and for currencies. You must set the value to `languages` or `currencies`, respectively. That attribute is required for a localized editor component. Please avoid configuration that will render your code invalid. For example, a multi-line currency editor is an example of configuration that is not supported.
```

Comment:
In general, don't use "please" in instructional documentation unless you ask the reader to do something unusual, such as viewing a third-party website for technical support.